pkg/scheduler/plugins/drf/hdrf_test.go

```diff
@@ -252,7 +252,7 @@ func TestHDRF(t *testing.T) {
 
 		binder := &util.FakeBinder{
 			Binds:   map[string]string{},
-			Channel: make(chan string),
```

Comment:
Why change to 300?

Reply:
To avoid blocking binding in FakeBinder. Please review PR#3343 first.